api/useQuerySchemas.ts

```diff
@@ -1,19 +1,27 @@
-import { useQuery } from "@tanstack/react-query";
+import { useQuery, UseQueryOptions } from "@tanstack/react-query";
 import axios from "axios";
 
 const DIGITAL_PLANNING_DATA_SCHEMAS_JSON_URL =
-  "https://theopensystemslab.github.io/digital-planning-data-schemas/v0.7.0/schema.json";
+  "https://theopensystemslab.github.io/digital-planning-data-schemas";
 
-const useQuerySchemas = () => {
+// TODO: type the version more strictly
+const useQuerySchemas = (
+  version: string,
+  options?: Readonly<Omit<UseQueryOptions<unknown>, "queryKey" | "queryFn">>
+) => {
   const fetchFn = async (url: string) => {
     const response = await axios.get(url);
     return response.data;
   };
 
   return useQuery({
-    queryKey: ["schemas"],
-    queryFn: () => fetchFn(DIGITAL_PLANNING_DATA_SCHEMAS_JSON_URL),
+    queryKey: ["schemas", version],
+    queryFn: () =>
+      fetchFn(
+        `${DIGITAL_PLANNING_DATA_SCHEMAS_JSON_URL}/${version}/schema.json`
```

Comment:
So something tricky to highlight here - since the latest release (v0.7.1), this URL now accounts for a `/schemas` subdirectory and each released version may have multiple schemas published within in (eg `application.json`, `prototypeApplication.json`, `preApplication.json`, `decision.json` and so on).

In my mind, the meaningful things to toggle here are:
- Which schema: Application, PreApplication, PrototypeApplication, so on
- Which version: "latest" or "next" only? 
  - eg this would have tech benefit of letting us deal with same base URL then and just sort of ignore very early versions, which probably aren't as important to visually document? Main purpose of this viewer in my mind is to share with consumers to help explain what they can recieve _now_ and what changes are queued up! Older version in `/dist` are nice audit history but no longer in active use for any integrations!
  - I know we need to smooth out `latest` & `@next` dist folders though! See https://github.com/theopensystemslab/digital-planning-data-schemas/issues/242)